src/helpers/contracts/getLedgerRecord.ts

```diff
@@ -0,0 +1,6 @@
+export default function (original: string, derivative: string) {
```

Comment:
this function and `getEmailLedgerRecord.ts` from above looks very similar, do we actually need two different functions for this?

Reply:
Updated. Thank you!